### ISSUE TITLE
[Bug] Visit link missing port

### ISSUE THREAD

**Issue by reedy** [tomaae/homeassistant-mikrotik_router#385]
## Describe the issue

The router I have setup is set on a public IPv4 address, on a port that isn't port 80.

On `config/devices/device/`, the visit link seems to not use that port

## How to reproduce the issue

Create a config that includes a non default port (ie not port 80), and try and use the visit link to get to the web UI of the 'tik device

## Expected behavior

The visit link to include the correct port...

## Screenshots

![Screenshot 2024-11-27 at 21 08 48](https://github.com/user-attachments/assets/851cce9b-15ef-46a7-9af1-63fe99d3a5fe)

## Software versions
 - Home Assistant version: 2024.11.3
 - Mikrotik Router integration version: 2.1.4
 - Mikrotik Hardware: SXT R (not specific)
 - RouterOS version: 7.16.1

**Comment by reedy** [comment]
Still an issue..

**Comment by reedy** [comment]
This stale bot is far too eager.

**Comment by tomaae** [comment]
I agree, this new stale bot is annoying. But old well working one want obsolete in favor of this one.
Can you submit your diagnostic data? I'm mostly interested in setup section. Data from device are not necessary.

**Comment by reedy** [comment]
> Can you submit your diagnostic data? I'm mostly interested in setup section. Data from device are not necessary.

Just the json from "Download Diagnostics"? Or something after following https://github.com/tomaae/homeassistant-mikrotik_router?tab=readme-ov-file#enabling-debug ?

**Comment by tomaae** [comment]
download diagnostics
just "custom_components>data>entry" is enough:
```
  "data": {
    "entry": {
      "data": {
        "host": "192.168.0.1",
        "name": "myrouter",
        "password": "**REDACTED**",
        "port": 0,
        "ssl": true,
        "username": "**REDACTED**"
      },
```

**Comment by reedy** [comment]
```
  "data": {
    "entry": {
      "data": {
        "host": "81.xxx.xxx.xxx",
        "name": "Mikrotik",
        "password": "**REDACTED**",
        "port": 8728,
        "ssl": false,
        "username": "**REDACTED**"
      },
      "options": {}
    },
```

**Comment by reedy** [comment]
...
